tools/lkl/lib/nt-host.c

```diff
@@ -1,3 +1,4 @@
+#define _WIN32_WINNT 0x0600
```

Comment:
Wouldn't it be better to put this into the Makefile? I am not very familar with Windows-programming, but maybe there is a use case to compile against a newer API if one desires so?